rednotebook/util/markup.py

```diff
@@ -50,7 +50,6 @@
 TABLE_HEAD_BG = '#aaa'
 
 CHARSET_UTF8 = '<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />'
-PRINT_FUNCTION = '<script></script>'
```

Comment:
At least when I added it, this empty script block was important for MathJax to work correctly. have you checked that this block is unneeded now?

Reply:
I have just tested LaTeX rendering with current versions of Firefox and Chrome - it works without those tags. Maybe that was a web browser quirk?